react/src/pages/SessionLauncherPage.tsx

```diff
@@ -639,7 +639,14 @@ const SessionLauncherPage = () => {
               form={form}
               layout="vertical"
               requiredMark="optional"
-              initialValues={INITIAL_FORM_VALUES}
+              initialValues={_.merge(
+                INITIAL_FORM_VALUES,
+                baiClient._config?.default_session_environment && {
+                  environments: {
+                    version: baiClient._config?.default_session_environment,
+                  },
+                },
```

Comment:
`INITIAL_FORM_VALUES` is used [to determine whether to apply form values](https://github.com/lablup/backend.ai-webui/pull/2337/files#diff-cb5aa3700f469823d540634a1acf2796ffcfa96416d6fa9333bfab0ff658b2c1R232-R243) using query parameters. Therefore, `INITIAL_FORM_VALUES` should be the initial values for a `<Form/>` without modification. Let's create a function outside of the component to handle the default image of `config.toml`.
`getInitialFormValues()` instead of `_.merge(INITIAL_FORM_VALUES...`

Reply:
In order to use "config" settings from `config.toml`, I need to use baiClient hook, which can be called only inside of React function. Therefore, instead of creating a function like `getInitialFormValues()` I would like to rearrange `INITIAL_FORM_VALUES` inside React function with default_session_environment value.